docs/configuration-validation.md

```diff
@@ -27,3 +27,13 @@ Add `configValidationMode` to your service configuration with one of the followi
 | error | Fail invoked command and output configuration errors. |
 | warn  | Output configuration errors in form of warnings.      |
 | off   | Suppress configuration errors.                        |
+
+## example of serverless.yml
+```
+service: my-project
+frameworkVersion: '3'
+configValidationMode: 'error'
+
+provider:
+  name: aws
+```
```

Comment:
Let's simply it to:

````markdown
_e.g. in serverless.yml:_
```
configValidationMode: 'error'
```
````

Reply:
commit updated,
simplified, but kept "service" to better see the level